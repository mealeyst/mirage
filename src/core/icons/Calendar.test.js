import React from 'react'
import { css } from 'styled-components'
import 'jest-styled-components'

import { Calendar } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) Calendar', () => {
  const createCalendar = (props) => {
    return shallowWithTheme(<Calendar {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createCalendar())
    .toMatchSnapshot()
  })
})
