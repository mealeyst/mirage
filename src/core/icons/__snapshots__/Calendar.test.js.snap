// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) Calendar matching the snapshot 1`] = `
.c0 path {
  fill: #0073D1;
}

<Component
  className="c0"
/>
`;
